File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int actualSum = 0 ; 
        int size = nums.size();
        int expectedSum = size*(size+1)/2;

        for(int num : nums){
            actualSum+= num ; 
        }

        return expectedSum-actualSum ; 
    }
};

File: 0268-missing-number/0268-missing-number.cs
public class Solution {
    public int MissingNumber(int[] nums) {
        int sum = 0 ;
        int size = nums.Length ;
        for(int i = 0 ; i < size; i++){
            sum+=nums[i];
        }
        int totalSum =  ((size)*(size+1))/2;
        return totalSum - sum ;
    }
}